python/ray/autoscaler/commands.py

```diff
@@ -377,7 +375,7 @@ def exec_cluster(config_file,
     Arguments:
         config_file: path to the cluster yaml
         cmd: command to run
-        docker: whether to run command in docker container of config
+        run_env: whether to run the command on the host or in a container
```

Comment:
make a comment about the possible options?

Reply:
OK!